protos/sources/vega/vega.proto

```diff
@@ -176,10 +176,11 @@ message PeggedOrder {
 
 // Details of an iceberg order
 message IcebergOrder {
-  // Size of the order that will be made visible if the iceberg order is refreshed after a trade has occurred.
-  uint64 initial_peak_size = 1;
-  // Threshold at which the order's visible remaining size will be refreshed back to its initial peak size.
-  uint64 minimum_peak_size = 2;
+  // Size of the order that will be made visible if the iceberg order is replensished after trading.
+  uint64 peak_size = 1;
+  // If the visible size of the order falls below this value it will be replensished back to the peak size using the 
+  // reserved amount back to the peak size.
```

Comment:
```suggestion
  // If the visible size of the order falls below this value, it will be replenished back to the peak size using the reserved amount.
```